recipes/filesendercli/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "FileSenderCLI" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/madsi1m/FileSenderCLI/releases/download/{{ version }}/{{ name }}.tar.gz
+  sha256: 2dcabb8007769d2488e6c7055d49e273a1496d1b9811b67a0a3e361f2c97a308
+
+build:
```

Comment:
It looks like this package can be `noarch: python`. See:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

It looks like there is a console script entry point. Please add it along with a test:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

Reply:
i removed the `noarch: python` as this was breaking the windows build (that i don't need)

Reply:
I don't exactly know what to put as an entrypoint as the script doesn't have a ::main

Reply:
added a test, calling `filesender -h`

Reply:
worked out the entrypoint :)